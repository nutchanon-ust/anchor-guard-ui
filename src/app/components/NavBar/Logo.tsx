import * as React from 'react';
import styled from 'styled-components/macro';

export function Logo() {
  return (
    <Wrapper>
      <Title>React Boilerplate</Title>
      <Description>Create React App Template</Description>
    </Wrapper>
  );
}

const Wrapper = styled.div`
  display: flex;
  align-items: center;
`;

const Title = styled.div`
  font-size: 1.25rem;
  color: ${p => p.theme.text};
  font-weight: bold;
  margin-right: 1rem;
`;

const Description = styled.div`
  font-size: 0.875rem;
  color: ${p => p.theme.textSecondary};
  font-weight: normal;
`;
