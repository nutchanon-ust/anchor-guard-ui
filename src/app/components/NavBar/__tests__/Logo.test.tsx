import * as React from 'react';
import { render } from '@testing-library/react';
import { Logo } from '../Logo';

describe('<Logo />', () => {
  it('should match snapshot', () => {
    const logo = render(<Logo />);
    expect(logo.container.firstChild).toMatchSnapshot();
  });
});
