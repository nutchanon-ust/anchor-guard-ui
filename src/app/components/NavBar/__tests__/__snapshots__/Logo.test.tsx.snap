// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  font-size: 1.25rem;
  font-weight: bold;
  margin-right: 1rem;
}

.c2 {
  font-size: 0.875rem;
  font-weight: normal;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    React Boilerplate
  </div>
  <div
    class="c2"
  >
    Create React App Template
  </div>
</div>
`;
